frontend/src/components/common/form/ParentSelect.scss

```diff
@@ -0,0 +1,14 @@
+.map-filter-bar {
+  .agency-item {
+    .form-control {
+      width: 230px;
+    }
+    .form-group {
+      .rbt {
+        .rbt-menu {
+          width: 370px !important;
```

Comment:
think I need the `!important` unfortunately

Reply:
pity, I'm not good enough at css to solve it either.